src/test/java/guitests/ChromeDriverTest.java

```diff
@@ -145,7 +145,7 @@ public void chromeDriverStubTest() {
         clearKeyCode();
 
         // jump to comments
-        push(KeyCode.C);
+        push(KeyCode.R);
```

Comment:
Realised that the constants aren't used much in tests. Unless we can `push` a `KeyCombination`...